test_api_wiki_call.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+from unittest import TestCase
+from unittest.mock import patch
+
+import API_Wiki_Call
+
+class TestAPI_Wiki_call(TestCase):
+
+    @patch('requests.Response.json')
+    def test_api_wiki_call_data_returned(self, mock_wiki_json):
+        mock_user_date = 'October 29'
+        example_wiki_api_response = ['October 29', ['October 29'], [''], ['https://en.wikipedia.org/wiki/October_29']]
+        mock_wiki_json.side_effect = [example_wiki_api_response]
+        wiki_api_response = API_Wiki_Call.API_Wiki_Call(mock_user_date)
+        self.assertTrue(mock_wiki_json, wiki_api_response.data)
+
+    @patch('requests.Response.json')
+    def test_api_wiki_call_user_error(self, mock_wiki_jason):
+        mock_user_error = 'completegibberishtogetusererror'
+        example_wiki_api_response = [mock_user_error, [],[],[]]
+        mock_wiki_jason.side_effect = [example_wiki_api_response]
+        wiki_api_response = API_Wiki_Call.API_Wiki_Call(mock_user_error)
+        self.assertTrue(mock_wiki_jason, wiki_api_response.user_error)
```

Comment:
Same as previous comment

Reply:
I fixed this as well, I'm currently finishing it up.